src/main/xsl/preprocess/topicpullImpl.xsl

```diff
@@ -253,35 +353,41 @@ mode="topicpull:figure-linktext" and mode="topicpull:table-linktext"
         <xsl:copy>
           <xsl:apply-templates select="@*"/>
           <!--create variables for attributes that will be passed by parameter to the getstuff template (which is shared with link, which needs the attributes in variables to save doing inheritance checks for each one)-->
-          <xsl:variable name="type" as="xs:string">
+          <xsl:variable name="type" as="xs:string?">
```

Comment:
Since we now return `()` for the else-branch, we can just remove the whole `xsl:choose` and just use `<xsl:variable name="type" select="@type" as="xs:string?"/>`